.github/workflows/e2e.yml

```diff
@@ -115,6 +115,8 @@ jobs:
                   # Recommended: pass the GitHub token lets this action correctly
                   # determine the unique run id necessary to re-run the checks
                   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+                  # Tolerate flaky tests
+                  CYPRESS_RETRIES: 3
```

Comment:
Shame that this is needed.

Reply:
In my experience, this is somewhat typical in tests that deal with UI, but agreed.